hosting/setting-up-hostd/linux/debian.md

```diff
@@ -31,7 +31,7 @@ This guide will walk you through setting up `hostd` on Linux. At the end of this
 
 To ensure you will not run into any issues with running `hostd` it is recommended your system meets the following requirements:
 
-* **Network Access:** `hostd` needs a stable internet connection and open network access in order to store and retrieve data on the Sia network.
+* **Network Access:** `hostd` needs a stable internet connection and open network access in order to store and retrieve data on the Sia network. You will also need to forward the ports `9981-9983` so `hostd` can properly communicate with the network and renters.
```

Comment:
it dosen't need 9984 too?

Reply:
that question applies to all files excluding [hosting/setting-up-hostd/macos.md](https://github.com/SiaFoundation/docs/pull/97/files#diff-961f85d35661e4703f599a98763a14b73d38a2a99c5ed19d65937ad9e82e7502)